inc/Engine/Admin/Settings/Subscriber.php

```diff
@@ -1,18 +1,18 @@
 <?php
-namespace WP_Rocket\Subscriber\Admin\Settings;
+namespace WP_Rocket\Engine\Admin\Settings;
 
+use Imagify_Partner;
 use WP_Rocket\Event_Management\Subscriber_Interface;
-use WP_Rocket\Admin\Settings\Page;
 
 defined( 'ABSPATH' ) || exit;
 
 /**
- * WP Rocket settings page subscriber
+ * WP Rocket settings page subscriber.
  *
+ * @since 3.6 Moves into the new architecture.
```

Comment:
```suggestion
 * @since 3.5.5 Moves into the new architecture.
```

Possibly it's not even needed